### DISCUSSION TITLE
Using Velero in a k8s cluster that uses gitops - are there any best practices ?

### DISCUSSION THREAD

**Discussion by harrycain72** [vmware-tanzu/velero#3827] [Community support Q&A]
We have an AKS cluster in place where eveything is installed via gitops using ArgoCD.
Velero works like a charm and everything would be in great order if we wouldn't have gitops in place.

We are looking for a way to backup only the pvc as all other objects are under control of gitops.
Is there a best practice how to deal with this topic ?
Is  Velero able to backup ONLY persistent volumes ?

Roland

**Comment by amrap030** [comment]
@harrycain72 I have basically the same question, did you found a solution or strategy for combining gitops and velero?

**Comment by blackpiglet** [comment]
By far, there is no way to only back up the data in the volume yet.

**Reply by kaovilai** [reply]
https://github.com/vmware-tanzu/velero/pull/7481 that would fit into this design IIUC.
